owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/shares/GetRemoteShareesOperation.java

```diff
@@ -46,64 +47,70 @@
 
 /**
  * Created by masensio on 08/10/2015.
- * <p>
+ *
  * Retrieves a list of sharees (possible targets of a share) from the ownCloud server.
- * <p>
+ *
  * Currently only handles users and groups. Users in other OC servers (federation) should be added later.
- * <p>
+ *
  * Depends on SHAREE API. {@See https://github.com/owncloud/documentation/issues/1626}
- * <p>
+ *
  * Syntax:
- * Entry point: ocs/v2.php/apps/files_sharing/api/v1/sharees
- * HTTP method: GET
- * url argument: itemType - string, required
- * url argument: format - string, optional
- * url argument: search - string, optional
- * url arguments: perPage - int, optional
- * url arguments: page - int, optional
- * <p>
+ *    Entry point: ocs/v2.php/apps/files_sharing/api/v1/sharees
+ *    HTTP method: GET
+ *    url argument: itemType - string, required
+ *    url argument: format - string, optional
+ *    url argument: search - string, optional
+ *    url arguments: perPage - int, optional
+ *    url arguments: page - int, optional
+ *
  * Status codes:
- * 100 - successful
+ *    100 - successful
  *
```

Comment:
Why do you add those spaces?

Reply:
https://github.com/owncloud/android-library/pull/224 apply a proper code style for all the files and I did not apply the proper style in here, good catch, I've just already fixed it.